### DISCUSSION TITLE
NullReferenceException: Object reference not set to an instance of an object.

### DISCUSSION THREAD

**Discussion by Dreitser** [jeffreylanters/react-unity-webgl#278] [Help Wanted!] [answered]
This repo is amazing in every way! It's so useful. 
Communication from Unity to React works perfectly! 
Communication from React to Unity there are some errors that we're stuck on.
Using Unity ver 2019.4.29f1 LTS and ver 7 of react-unity-webgl
we're getting the following errors in the web console. 
>index.js:1 NullReferenceException: Object reference not set to an instance of an object.
(Filename: currently not available on il2cpp Line: -1)
console.<computed> @ index.js:1
>index.js:1 MissingMethodException: The method 'System.Void GameController::WalletConnected(System.String)' has no implementation.
(Filename: currently not available on il2cpp Line: -1)
We've tried going through all versions of the documentation and not making any progress.

**Comment by jeffreylanters** [comment, marked as answer]
Hi! Looking at your error, it looks like the method you're trying to invoke is not available publicly. Make sure the method you're trying to invoke is available via a script which is attached to a game object with the name matching the first parameter passed to the send method in your React application.

Let me know if this helps you. If you're unable to resolve this problem, feel free to reply on this thread. Some example code would be helpful as well.

Best of luck!

**Reply by Dreitser** [reply]
So the Send method works perfectly!
This error occurs strangely enough on the receive call (Unity -> React) 
In React, listeners are set up and in Unity everything looks right.

**Reply by Dreitser** [reply]
This from Unity helped me resolve the issue. Now it works like a charm.
https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html
